### PR TITLE
feat(ui-components) Add Input Component

### PR DESCRIPTION
**Pull request by robertpin**
Remove unused component

### CODE REVIEW

**Comment by saghul** [comment]
Doesn't the native participants pane have a similar control?

**Comment by robertpin** [comment]
> Doesn't the native participants pane have a similar control?

It does. We need to create a new component for native as well

File: react/features/participants-pane/components/native/styles.js [hunk 1]
```diff
@@ -310,5 +310,16 @@ export default {
...
+    },
+
+    inputContainer: {
+        marginLeft: BaseTheme.spacing[3],
+        marginRight: BaseTheme.spacing[3],
+        marginBottom: BaseTheme.spacing[4]
+    },
+
+    centerInput: {
+        paddingRight: 16,
```
Comment (Calinteodor) [inline react/features/participants-pane/components/native/styles.js:322]: Use tokens.

File: react/features/base/react/components/native/inputStyles.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    inputFocused: {
+        borderColor: BaseTheme.palette.focus01
+    },
+
+    inputError: {
+        borderColor: BaseTheme.palette.textError
+    },
+
+    iconInput: {
+        paddingLeft: 40
```
Comment (Calinteodor) [inline react/features/base/react/components/native/inputStyles.ts:54]: Tokens.

File: react/features/base/react/components/native/inputStyles.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    },
+
+    clearableInput: {
+        paddingRight: 40
```
Comment (Calinteodor) [inline react/features/base/react/components/native/inputStyles.ts:58]: Tokens.

File: react/features/base/react/components/native/inputStyles.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        zIndex: 1,
+        top: 13,
+        left: 16
+    },
+
+    input: {
+        backgroundColor: BaseTheme.palette.ui03,
+        color: BaseTheme.palette.text01,
+        paddingVertical: 13,
+        paddingHorizontal: 16,
```
Comment (Calinteodor) [inline react/features/base/react/components/native/inputStyles.ts:32]: Tokens.

**Comment by robertpin** [comment]
> @robertpin @saghul Based on the fact that we are replacing ClearableInput with the new Input component inside MeetingParticipantList, we can do the same inside AddPeopleDialog, SpearkerStatsSearch and GifsMenu. Aside from that, everything fits like a glove!

Out of scope. The purpose of this PR is just to introduce the new components. We used them in 1 instance si that we don't add unused code. The other Inputs will be replaces in subsequent PRs

**Comment by saghul** [comment]
No problem, as long as we use it in 1 place to make sure we have covered the use case, we can replace them in subsequent PRs.

**Review by Calinteodor — APPROVED** [review]

**Pull request merged**